pkgs/development/tools/azcopy/default.nix

```diff
@@ -0,0 +1,24 @@
+# with import <nixpkgs>{};
+{ stdenv, fetchFromGitHub, buildGoPackage }:
+
+buildGoPackage rec {
+  name = "azure-storage-azcopy-${version}";
+  version = "10.0.1";
```

Comment:
I this this should have `pre` in it, since they are listed as prereleases on GitHub.